tracing-subscriber/src/filter/env/field.rs

```diff
@@ -119,6 +233,20 @@ impl PartialOrd for ValueMatch {
     }
 }
 
+impl fmt::Display for ValueMatch {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match &self.inner {
+            ValueMatchInternal::Bool(inner) => fmt::Display::fmt(inner, f),
+            ValueMatchInternal::F64(inner) => fmt::Display::fmt(inner, f),
+            ValueMatchInternal::NaN => fmt::Display::fmt(&f64::NAN, f),
+            ValueMatchInternal::I64(inner) => fmt::Display::fmt(inner, f),
+            ValueMatchInternal::U64(inner) => fmt::Display::fmt(inner, f),
+            ValueMatchInternal::Debug(inner) => fmt::Display::fmt(inner, f),
+            ValueMatchInternal::Pat(inner) => fmt::Display::fmt(inner, f),
+        }
+    }
+}
+
```

Comment:
Pulled up so that after this PR all `impl ValueMatch` are under the same `==== impl ValueMatch ===` section.